tests/unit/test_util.py

```diff
@@ -25,3 +26,43 @@ def test_invalid_cfg_file(mock_extra_settings_file):
     """
     with pytest.raises(configparser.MissingSectionHeaderError):
         load_cfg(mock_extra_settings_file)
+
+
+class FakeModel(BaseApiModel):
+    name: str
+
+
+class TestParseInput:
```

Comment:
I am not 100% decided, but I like this direction for grouping related tests together, so I was experimenting with it (I used this same structure in the service version of these tests, fwiw). Personally, I sometimes find it less clear exactly what it under test than I think it could be with test naming only. The idea here is to group related tests to help make it more obvious and easier to dig into issues as they arise.